dev-infra/format/formatters/prettier.ts

```diff
@@ -27,8 +27,11 @@ export class Prettier extends Formatter {
    * The configuration path of the prettier config, obtained during construction to prevent needing
    * to discover it repeatedly for each execution.
    */
-  private configPath =
-      this.config['prettier'] ? exec(`${this.binaryFilePath} --find-config-path .`).trim() : '';
+  private configPath = this.config['prettier'] ?
+      spawnSync(this.binaryFilePath, ['--find-config-path', '.'], {
+        encoding: 'utf8'
```

Comment:
You probably want to use `shell: true`, otherwise this will not work on Windows unless `binaryFilePath` contains the `.cmd` extension:

```suggestion
        encoding: 'utf8',
        shell: true,
```